packages/app/src/cli/services/generate.ts

```diff
@@ -46,12 +46,12 @@ async function generate(options: GenerateOptions) {
   const app: AppInterface = await loadApp({
     directory: options.directory,
     configName: options.configName,
-    specifications,
+    ...specifications,
   })
-  const availableSpecifications = specifications.map((spec) => spec.identifier)
+  const availableSpecifications = specifications.specs.map((spec) => spec.identifier)
```

Comment:
um, this is weird and will be weirder if we call the normal specs just `specifications` 🤔 
We maybe need a new name either for the object containing both types or the non-config ones:
```ts
allSpecifications: { specifications: ..., configSpecifications: ...}
// or
allSpecifications: { extensionSpecifications: ..., configSpecifications: ...}
// or
specifications: { extensionSpecifications: ..., configSpecifications: ...}
```